test/gpu/native/setBlockSizePrimitive.execopts

```diff
@@ -0,0 +1 @@
+--verbose
```

Comment:
I'm OK with this. But I want to get away from this as much as possible. My https://github.com/chapel-lang/chapel/pull/20186 has a commit that stops this (ab)use of `--verbose`. Not sure the if we'll merge that PR, but even if we don't I'm inclined to cherry-pick that to a separate PR.

In case you aren't under too much pressure with other priorities, I suggest we tap into the `startVerboseGPU` interface to handle this and stop adding more technical debt. The change should be simple:

- Modify this macro [here](https://github.com/chapel-lang/chapel/blob/main/runtime/include/chpl-gpu-diags.h#L141-L144) to append something like "... with block size X"
- Pass the block size while "calling" that macro [here](https://github.com/chapel-lang/chapel/blob/main/runtime/src/chpl-gpu.c#L223).

Reply:
I've updated GPUDiagnostics to output size (see commit: 9d559a8ac3b726ecaf63d09af50ae5ddac2eeafc)